api/net/Areas/Editor/Controllers/StorageController.cs

```diff
@@ -575,5 +614,20 @@ public async Task<IActionResult> UploadFilesToS3([FromQuery] DateTime? updatedBe
             FailedUploads = failedUploads,
         });
     }
+
+    /// <summary>
+    /// Delete old local files
+    /// </summary>
+    /// <param name="beforeDate">only delete files updated before the specified date</param>
+    /// <returns>deleted file count</returns>
+    [HttpDelete("delete-old-local-files")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(int), (int)HttpStatusCode.OK)]
+    [SwaggerOperation(Tags = new[] { "Storage" })]
+    public async Task<IActionResult> DeleteOldLocalFiles([FromQuery] DateTime beforeDate)
```

Comment:
Standard convention for async functions are to rename to `DeleteOldLocalFilesAsync`